src/doc/App.js

```diff
@@ -1,124 +1,16 @@
 import React from 'react';
-import {
-  createMuiTheme,
-  ThemeProvider,
-  makeStyles,
-} from '@material-ui/core/styles';
+import { ThemeProvider, makeStyles } from '@material-ui/core/styles';
 import Container from '@material-ui/core/Container';
 import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
-import Header from './components/Header';
+import { geopsTheme, Header, Footer } from '@geops/geops-ui/lib';
```

Comment:
I'm wondering why eslint throws this error
`unable to resolve path to module`
when using
`import { geopsTheme, Header, Footer } from '@geops/geops-ui';` without `/lib`
although `yarn start` works.
See https://gitlab.geops.de/geops/geops-ui/-/blob/eva/init/package.json#L5

Reply:
@gevos Should be fixed with https://github.com/geops/mobility-toolbox-js/pull/34/commits/8a3643953915fdbdeeaa2f4543445e6c21517ce4